src/os/bluestore/BlueRocksEnv.cc

```diff
@@ -99,13 +99,13 @@ class BlueRocksRandomAccessFile : public rocksdb::RandomAccessFile {
 
   // Used by the file_reader_writer to decide if the ReadAhead wrapper
   // should simply forward the call and do not enact buffering or locking.
-  bool ShouldForwardRawRequest() const override {
```

Comment:
@tchaikov Right, but for some reason the env.h in my environment appears not having those functions. I will close this PR.